nss-day-cohort-43-csharp/MVC-Blue-Barracudas
# Bl delete comment

Testing:
- Navigate to comments page of a post
- Click on the delete link for one of the comments which should navigate to the delete page for that comment
- Clicking the delete button should remove the comment from the database and go back to the list of comments
- Clicking the back button should navigate back to the list without deleting the comment
